addons/web/static/src/views/kanban/kanban.variables.scss

```diff
@@ -0,0 +1,25 @@
+$o-kanban-default-record-width: 300px !default;
+$o-kanban-small-record-width: 240px !default;
```

Comment:
When moving code between files, it's always better to create a specific `[MOV]`commit that does just that and nothing else. In almost all the cases Git is smart enough to understand what you've done, but it's not always the case.

Reply:
It’s already the case, or am I missing something ? 😅 
https://github.com/odoo/odoo/pull/96195/commits/ab17e8999cc06aa40d3899833055b62d393816ab

Reply:
sorry, my bad , good 👍